lib/elixir_boilerplate_web/endpoint.ex

```diff
@@ -80,6 +80,12 @@ defmodule ElixirBoilerplateWeb.Endpoint do
     end
   end
 
+  defp cors(conn, _opts) do
+    opts = Corsica.init(Application.get_env(:elixir_boilerplate, Corsica))
+
+    Corsica.call(conn, opts)
+  end
+
```

Comment:
@remiprev As for the reordering here, I use the same order we have in the Plug lifecycle…

```elixir
plug(ElixirBoilerplateWeb.Health.Plug)
  plug(:canonical_host)
  plug(:force_ssl)
  plug(:cors)
  plug(:basic_auth)
  plug(:session)
```